app/src/interfaces/map/index.ts

```diff
@@ -63,32 +134,13 @@ export default defineInterface({
 				width: 'half',
 				interface: 'dropdown',
 				options: {
-					choices: Object.keys({ ...mapbox_sources, ...sources }).map((choice) => ({ value: choice, text: choice })),
+					choices: getBasemapSources().map((b) => ({ value: b.name, text: b.name })),
```

Comment:
Note to self: This is the bit that relies on `directus_settings` which isn't populated yet at this point in time.